### DISCUSSION TITLE
Changing Fontsize in CSS

### DISCUSSION THREAD

**Discussion by Kurloc** [Textualize/textual#1308] [Q&A] [answered]
Hello there, I'm using textual 0.5 to build out a TUI for my chess engine. I work w/ 4k monitors most of the time, so I was trying to increase font size using CSS. 

I was looking through the docs and I don't see the font-size keywords mentioned. I tried quite a few ways in the CSS w/o success, and I was wondering is this possible at this point in them?

**Comment by davep** [comment, marked as answer]
The choice and size of font is something that's controlled by your terminal, not by a Textual application.

**Reply by Kurloc** [reply]
Ah that makes sense, normally I can ctrl scroll in terminal previewer on windows to change the font size, but it seems to have no effect while I was running my textual app. I'll try some more terminals, thank you!
